docs/source/user/api.rst

```diff
@@ -353,7 +366,7 @@ Login (Obtain User Auth Token)
 
 .. code-block:: text
 
-    /api/v1/account/token/
+    /api/v1/<organization-slug>/account/token/
```

Comment:
Not, I noticed this did not include `<organization-slug>` but in usage, I was putting <org-slug> to get the token, I assume this one only a typing mistake.